tests/pytest/conftest.py

```diff
@@ -1,10 +1,15 @@
+DEFAULT_MODEL_IMAGE = 'us.gcr.io/vcm-ml/fv3gfs-compiled'
+
 
 def pytest_addoption(parser):
     parser.addoption(
         "--refdir", action="store", default="reference/docker", help="directory for reference files"
     )
     parser.addoption(
-        "--image_tag", action="store", default="default", help="The docker image tag to run"
+        "--image_version", action="store", default="latest", help="The docker image version to run"
+    )
+    parser.addoption(
+        "--image", action="store", default=DEFAULT_MODEL_IMAGE, help="The docker image name to run, without tags"
     )
     parser.addoption(
         "--code_root", action="store", default="/", help="The path to the codebase to test"
```

Comment:
Should we remove this now that the serialized code path is the same as the default? Looks like the fixture was deleted.

Reply:
The fixture's still there, it's used to modify the base part of the image. The used images are `<--image>:<--image_version>-serialize` and `<--image>:<--image_version>`. We don't really have a use case for it at the moment so I could remove it, but it seems clean enough that it shouldn't hurt? Mostly if you build an image with a custom name, this allows you to run it through the regression tests.